tests/release/README.md

```diff
@@ -58,8 +58,17 @@ This document will detail the e2e tests included for the release suite.
        - The Release passes.
        - Validate that the SBOM for both components were uploaded to Pyxis stage.
 
+### 4. Ensure ReleasePlan has owner references set (e2e-test-releaseplan-owner-ref.go)
 
-### 5. Negative e2e-tests
+   This test is designed to ensure that the ReleasePlan controller reconciles ReleasePlans to have an owner reference for its application.
+
+      Steps:
+       - Create the a ReleasePlan and an Application.
+      Expected Results:
+       - The ReleasePlan has an owner reference for the Application.
+       - If the Application is deleted, the ReleasePlan is also deleted.
+
+### 6. Negative e2e-tests
```

Comment:
Also, need to update the sub-bullet points from 5.* to 6.*

Reply:
nice catch